cognito-pre-token-generation-openid/src/main/java/no/unit/nva/cognito/UserAccessRightForCustomer.java

```diff
@@ -22,7 +25,12 @@ public UserAccessRightForCustomer(CustomerDto customerDto,
         this.customer = customerDto;
     }
 
-    public static List<UserAccessRightForCustomer> fromUser(UserDto user, Set<CustomerDto> customers) {
+    public static List<UserAccessRightForCustomer> fromUser(UserDto user, Set<CustomerDto> customers,
+                                                            Boolean hasAcceptedTerms) {
```

Comment:
can it be boolean? I see you send `true` in UserSelectionUponLoginHandler, otherwise `false`?